index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="authors" content="Salman & Daniel" />
+    <title>Awesome Books</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <main>
+      <section id="firstBookContainer">
+        <div id="TitleContainer">
+          <h1 class="Title">Awesome Books</h1>
+          <ul id="bookList"></ul>
+          <form action="#">
+            <input type="text" id="title" placeholder="title" /><br /><br />
+            <input type="text" id="author" placeholder="author" /><br /><br />
+            <button type="button" id="addBtn">Add</button>
+          </form>
```

Comment:
- [ ] [OPTIONAL] Everything is fine about the projects 👏 , however I noticed that you when you are adding a book while the inputs are empty, your app is still adding a book, kindly make sure you add some input validations before adding 👍

Reply:
Thanks @iambenkis I have also add the validation as you suggested.